vowpalwabbit/cbify.cc

```diff
@@ -478,9 +478,25 @@ void output_example_seq(vw& all, multi_ex& ec_seq)
   }
 }
 
-void finish_example(vw& all, cbify&, example& ec)
+void output_example_regression(vw& all, cbify& data, example& ec)
+{
+  // data contains the cb_cont vector, which store among other things, loss
+  // ec contains a simple label type
+  label_data& ld = ec.l.simple;
+  const auto& cb_cont_costs= data.regression_data.cb_cont_label.costs;
+  if (cb_cont_costs.size() > 0)
```

Comment:
cb_cont_costs should have exactly 1 element in it (associated with the last example that predict_or_learn_regression was called with).

Reply:
Should be correct.

---
In reply to: [316680114](https://github.com/mmajzoubi/vowpal_wabbit/pull/10#discussion_r316680114) [](ancestors = 316680114)